src/main/scala-2.12/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -29,19 +29,23 @@ object JmapMessages {
   type JmapParameters = Map[String, Any]
   val NO_PARAMETERS : JmapParameters = Map()
 
+  val MessageIdSessionParam = "messageId"
+  val SubjectSessionParam = "subject"
+  val TextBodySessionParam = "textBody"
```

Comment:
no need to capitalize, it's not a type.

Reply:
you are right, but I took this convention from UserFeeder. If the last commit makes consensus, I will update every constant